stackit/internal/services/ske/cluster/resource.go

```diff
@@ -505,6 +505,10 @@ func (r *clusterResource) Schema(_ context.Context, _ resource.SchemaRequest, re
 			"network": schema.SingleNestedAttribute{
 				Description: "Network block as defined below.",
 				Optional:    true,
+				Computed:    true,
```

Comment:
What is it exactly that the API returns in this case then? I'm not sure if the field should be Computed as it's not really something that the API can set, right?

Reply:
Yeah you're right, I was mislead by the other nested field `maintenance`, which can actually be set by the API. I will update with another fix, similarly to the `extensions` field, which has the same type and is not `Computed`